biogeochem/EDPatchDynamicsMod.F90

```diff
@@ -301,6 +301,7 @@ subroutine spawn_patches( currentSite, bc_in)
     
     use EDParamsMod         , only : ED_val_understorey_death, logging_coll_under_frac
     use EDCohortDynamicsMod , only : zero_cohort, copy_cohort, terminate_cohorts 
+    use ChecksBalancesMod   , only : SiteCarbonStock
```

Comment:
seems like this isn't needed since all calls to it have been commented out?

Reply:
yeah.  Maybe I should remove the comments too.  I suppose there is no reason to keep them. Will update